packages/db/src/schema/page.ts

```diff
@@ -29,7 +29,7 @@ export const page = sqliteTable("page", {
 });
 
 export const pageRelations = relations(page, ({ many, one }) => ({
-  monitorsToPages: many(monitorsToPages),
+  monitors: many(monitorsToPages),
```

Comment:
I'd suggest to use `monitors`. This fits well for the relational queries from drizzle. Than, we don't have to access the values via `page.monitorsToPages`.

Reply:
but it's a many-to-many relation ship we need a join table

Reply:
https://orm.drizzle.team/docs/rqb#many-to-many 😭

Reply:
Thanks a lot! Reverted back! 🚀